src/core/pretty.ml

```diff
@@ -205,6 +205,13 @@ let pp_command : p_command pp = fun oc cmd ->
       List.iter (out " %a" pp_p_arg) args;
       Option.iter (out " :@ @[<hov>%a@]" pp_p_term) ao;
       out " ≔@ @[<hov>%a@]@]" pp_p_term t
+  | P_inductive(e,s,t,tl)   ->
+      out "@[<hov 2>%ainductive %a" pp_expo e pp_ident s;
+      Option.iter (out " :@ @[<hov>%a@] ≔@ " pp_p_term) t;
+      let pp_pair pp_a pp_b oc fmt (a,b) =
+        Format.fprintf oc fmt pp_a a pp_b b in
+      List.iter (pp_pair pp_ident pp_p_term oc
+                   ("\n |@ @[<hov>%a@] :@ @[<hov>%a@]")) tl
```

Comment:
Could be simplified as follows:
```
let pp_cons oc (id,a) = Format.fprintf "@[<hov>%a@] :@ @[<hov>%a@]" pp_ident id pp_p_term a in
List.pp pp_cons "\n| "
```

Reply:
Where is used the variable "oc" ?

Reply:
The code I write here is not type check; you should be able to fix by yourself. oc is the first argument of fprintf...

Reply:
```
let pp_cons oc (id,a) = Format.fprintf oc "@[<hov>%a@] :@ @[<hov>%a@]" pp_ident id pp_p_term a in
List.iter pp_cons "\n| " oc tl
```

Reply:
Done, thanks